larray/core/array.py

```diff
@@ -3104,10 +3114,15 @@ def argsort(self, axis=None, kind='quicksort'):
                 raise ValueError("array has ndim > 1 and no axis specified for argsort")
             axis = self.axes[0]
         axis = self.axes[axis]
-        pos = self.posargsort(axis, kind=kind)
+        pos = self.idxindirectsort(axis, kind=kind)
         return LArray(axis.labels[pos.data], pos.axes)
 
-    def posargsort(self, axis=None, kind='quicksort'):
+    def posargsort(self, axis=None):
+        """Deprecated
+        """
+        warnings.warn("posargsort has been replaced by idxargsort", FutureWarning, stacklevel=2)
```

Comment:
it needs to call the old method :)

Reply:
the new name is (according to the current state of this PR -- but as my comments show I think we can do better) idxindirectsort